openlibrary/plugins/worksearch/code.py

```diff
@@ -1,3 +1,6 @@
+import six
+from six.moves.urllib.parse import urlencode
```

Comment:
using `requests` should simplify this considerably.

Reply:
Strong agree; I looked into it, but it looked like a non-trivial addition to this change, so that's something we can do in a future PR. (A better solution might even be to use `solr_client` class to abstract these things so we don't manually build the url params). Closing as won't fix.